queue/queue.py

```diff
@@ -1,15 +1,19 @@
+import collections
+
 class Queue:
```

Comment:
Good call on the collections.dequeue though I think we were supposed to do it with a linked list. But to be fair this is probably the most efficient way to do it.